src/components/footer/footer.module.scss

```diff
@@ -32,3 +36,56 @@
     @include typography.body1;
   }
 }
+
+.bottomBar {
+  display: flex;
+  padding-bottom: 32px;
+  flex-direction: column;
+  justify-content: center;
+  align-items: flex-start;
+  gap: 32px;
+  @include fonts.poppins;
```

Comment:
what is this include here, include should be at the top if anything? and as far as I remember we do have a font family already specified somewhere in global scss file?